crates/kotlin-ffi/src/lib.rs

```diff
@@ -68,6 +78,72 @@ uniffi::custom_type!(FFIBytes, String, {
     lower: |obj| obj.to_string(),
 });
 
+#[derive(Debug, uniffi::Record)]
+pub struct RouteUiFields {
+    pub route: Vec<TxnDetails>,
+    pub bridge: Vec<TransactionFee>,
```

Comment:
I see that we have bridging_fee in `FundingMetadata`  Are these values the same?

Reply:
The values are the same, but the value returned from RouteUiFields contains more details: the value converted to local currency, etc.